### PR TITLE
Add contact section to the homepage

### PR DESCRIPTION
**Pull request by xialuna**
### Changes  includes:
- added a contact section to the home page
- removed  section className="min-h-screen" to the contact section
- import social media icons from phosphor icons

### CODE REVIEW

**Review by Lemon1903 — CHANGES_REQUESTED** [review]
Alright! The section loads very well in the homepage. The only thing that could be changed here is to extract the section from `pages/index.tsx` to a separate component in `components` folder. Please refer to how the `components\Home` are structured. For further details, refer to this [PR](https://github.com/PUP-The-Programmers-Guild/TPGWebsite/pull/31#pullrequestreview-1630087715)

File: src/pages/index.tsx [hunk 1]
```diff
@@ -33,9 +41,78 @@ export default function IndexPage() {
       <section className="min-h-screen">
         <h1 className="text-2xl underline">FAQ</h1>
       </section>
-
-      <section className="min-h-screen">
-        <h1 className="text-2xl underline">Contact Us</h1>
+      {/*CONTACT*/}
+      <section
+        className="z-100 text-white"
```
Comment (Lemon1903) [inline src/pages/index.tsx:46]: You could remove the class `z-100` since there is no tailwind class for that

**Comment by xialuna** [comment]
[Update]
### Changes includes:
- separate contacts from `index.tsx` and put it in the components folder
- remove z-100
- change size of icons and gap properties

**Review by Lemon1903 — CHANGES_REQUESTED** [review]
Ok it's nearly done, but there is one unresolved review from last review, please check it out. Let me know if there is a problem with that, thanks!

Then the final revision would be to move this code snippet in `index.tsx` into the `Contact.tsx` (please capitalize this as well for naming convention of component files).

from:
```tsx
<section
  className=" text-white"
  style={{
    background:
      "radial-gradient(100% 418.39% at 100% 100%, #343534 0%, #1E5C3F 33.33%, #409C49 66.67%, #FFEC5C 100%)",
  }}
>
  <Contacts />
</section>
```
to:
```tsx
<Contacts />
```
 So that the `Contacts.tsx` file would encapsulate all the code related to it.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -33,9 +41,78 @@ export default function IndexPage() {
...
+                <TwitterLogo color="#f8fafc" size={38} weight="fill" />
+              </a>
+              <a
+                href={process.env.NEXT_PUBLIC_LINK_GITHUB}
+                target="_blank"
+                rel="noreferrer"
+              >
+                <GithubLogo color="#f8fafc" size={38} weight="fill" />
+              </a>
+            </div>
```
[the remark this answers is not in the document]
Comment (xialuna) [reply src/pages/index.tsx]: [UPDATE]
### Changes Include
- Change naming convention of title from `contacts.tsx` to `Contacts.tsx`
- Reorganized the code within `Contacts.tsx` to encapsulate all related sections and styling.
- Used array of objects to manage the links and their respective icons.
- Used the map function to generate anchor elements for social media links

**Comment by xialuna** [comment]
[UPDATE]
### Changes Include
- Change naming convention of title from `contacts.tsx` to `Contacts.tsx`
- Reorganized the code within `Contacts.tsx` to encapsulate all related sections and styling.
- Used array of objects to manage the links and their respective icons.
- Used the map function to generate anchor elements for social media links

**Review by Lemon1903 — APPROVED** [review]
Just added a commit that refactors the Home folder structure and imports. Anyways, LGTM!

**Pull request merged**